### DISCUSSION TITLE
Issue with marketplace in the localhost.

### DISCUSSION THREAD

**Discussion by dejvidxh** [FlowiseAI/Flowise#3724] [Q&A]
Yo guys. I am having this issue whenever I try accessing http://localhost:3000/marketplaces It remains blank
![2](https://github.com/user-attachments/assets/6f4b0ec0-ceb3-4433-9327-b6acff61cd55)

I don't know how to code or anything. Here to play with AI Agents.

I got the Node.Js  latest version 23.4.0 and I followed the Quick start instructions.

Any idea how to fix this?

**Comment by nurdol26** [comment]
Got the same problem... and couldn't find any solution.  

Also, when I am trying to open "chat" bubbles, it makes the same screen as here, with this bug: 
<img width="728" alt="image" src="https://github.com/user-attachments/assets/0639d2ff-269f-4723-9c64-2626795b5775" />

**Reply by dejvidxh** [reply]
I resolved mine by updating Flowise with this command through node/js 

npm install -g flowise@2.2.1

nit sure but I guess this other command would work too 

npm update -g flowise@2.2.1

**Comment by infoinfy** [comment]
same problem 
![image](https://github.com/user-attachments/assets/f4cbd5f3-6af1-45e1-a72b-ec5524699a2c)

**Reply by juanicole1809** [reply]
For me the solution was to install flowise but not with npm instead with docker image... i am on macos

**Comment by AdarshStudiyo** [comment]
Downgrade your node version ( i have earlier 22.14.0 ) to 20.18.3 .
It work for me to watch marketplace in windows.
